apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/layout.tsx

```diff
@@ -11,14 +23,62 @@ import { getCurrentUser } from '$/services/auth/getCurrentUser'
 import { ROUTES } from '$/services/routes'
 
 import { Actions } from './_components/Actions'
-import { EvaluationResults } from './_components/EvaluationResults'
-import { MetricsSummary } from './_components/MetricsSummary'
+import Content from './_components/Content'
 
 const TYPE_TEXT: Record<EvaluationResultableType, string> = {
   [EvaluationResultableType.Text]: 'Text',
   [EvaluationResultableType.Number]: 'Numerical',
   [EvaluationResultableType.Boolean]: 'Boolean',
 }
+
+type ReturnType<T extends boolean> = T extends true
+  ? {
+      aggregationTotals: EvaluationAggregationTotals
+      meanOrModal: EvaluationMeanValue
+    }
+  : {
+      aggregationTotals: EvaluationAggregationTotals
+      meanOrModal: EvaluationModalValue
+    }
+async function fetchData<T extends boolean>({
+  workspaceId,
+  evaluation,
+  documentUuid,
+  isNumeric,
+  commit,
+}: {
+  isNumeric: T
+  workspaceId: number
+  evaluation: Evaluation
+  documentUuid: string
+  commit: Commit
+}): Promise<ReturnType<T>> {
+  const aggregationTotals = await getEvaluationTotalsQuery({
```

Comment:
are these methods cached? add the suffix if so

Reply:
Not cached. Should they be cached? It's only called once here in the server

Reply:
i always cache by default tbh, might not be necessary

Reply:
My rule of thumb is `import { cache } from 'react'` is like a Rails per request singleton. Like `current_user`. Only useful when is used more than once during that request.